lhotse/recipes/librimix.py

```diff
@@ -1,6 +1,6 @@
 import shutil
```

Comment:
This is a pre-existing issues, but I think all of these recipes should have some kind of file-level docstring to give some context.  The reader may not know, for instance, that librimix is a dataset based on Librispeech for speech separation.

Reply:
Great point!